README.md

```diff
@@ -22,6 +22,35 @@ cls.fit(X_train, y_train)
 predictions = cls.predict(X_test)
 ```
 
+## auto-sklearn acceleration with sklearnex
+
+You can accelerate auto-sklearn with [sklearnex](https://github.com/intel/scikit-learn-intelex). It is extension that speedups sklearn algorithms through patching with highly optimized machine learning kernels.
+
+Install sklearnex with pip or conda:
+```bash
+pip install scikit-learn-intelex
+conda install scikit-learn-intelex
+conda install -c conda-forge scikit-learn-intelex
+```
+
+Insert these two lines of patching code before auto-sklearn and sklearn imports to get acceleration:
```

Comment:
I suggest this version:
```suggestion
To accelerate auto-sklearn, patch scikit-learn before importing auto-sklearn and sklearn:
```

this is also ok:
```suggestion
To accelerate auto-sklearn, insert the following two lines of patching code before auto-sklearn and sklearn imports:
```

Reply:
Changed to 2nd suggestion